### PR TITLE
Solutions

### PR DESCRIPTION
**Pull request by marcusnaslund**
Here are the solutions, translated from imint.se and/or copied from the original versions on the front page.
I will extend them as I extend the solutions on imint.se.
@simonmika comments?

### CODE REVIEW

File: content/solutions/robots.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
++++
+title = "Ground Vehicles, UGVs and Robots"
+date = 2015-10-14T00:00:00Z
+image = "robotExplosiveDisposal.png"
+weight = 5
+[menu.main]
+parent = "Solutions"
+name = "Telefon"
```
Comment (simonmika) [inline content/solutions/robots.md]: Translate

File: content/solutions/underwater.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
++++
+title = "Underwater and Sub-Surface Video"
+date = 2015-10-14T00:00:00Z
+image = "uwROV.png"
+weight = 5
+[menu.main]
+parent = "Solutions"
+name = "Underwater and Sub-Surface Video"
```
Comment (simonmika) [inline content/solutions/underwater.md]: Just "Underwater" should be enough in the menu.

File: content/solutions/robots.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
++++
+title = "Ground Vehicles, UGVs and Robots"
```
Comment (davidhesselbom) [inline content/solutions/robots.md]: `UGVs, and Robots`

**Comment by marcusnaslund** [comment]
Updated. Should I mark them all as draft or how do we do this?

**Comment by simonmika** [comment]
No, let them go live when they are ready.

**Comment by simonmika** [comment]
Can we replace the social app image and the compression image?

**Comment by simonmika** [comment]
And ask @jonathanudd about the ROV image. What is the license?

File: content/solutions/aerialsurveillance.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+date = 2015-10-14T00:00:00Z
+image = "uavLaunch.png"
+weight = 4
+[menu.main]
+parent = "Solutions"
+name = "Aerial Surveillance and UAS"
++++
+Video from unmanned aerial systems can often be affected by disturbances such as uncontrollable movements, vibration, weather conditions, and more. The Vidhance® technology addresses this kind of issues and provides improved video to the viewer inside a ground control station or remote video terminal in real-time.
+
+This is something we have experience of ever since 2009.
```
Comment (davidhesselbom) [inline content/solutions/aerialsurveillance.md]: Sounds strange, as if it's not something we do every day. `experience with`, maybe? But that's not quite it, either...

**Comment by marcusnaslund** [comment]
Yes, we should do the same for imint.se.
I wasn't able to render the compression svg that you recommended. Could you fix it or do we have another?
I will talk to Charlotte (I can't reference her here..?) about a replacement for the social app image.

File: content/solutions/apps.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
++++
+title = "Apps and social media"
```
Comment (davidhesselbom) [inline content/solutions/apps.md]: `Apps and Social Media`

File: content/solutions/apps.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+date = 2015-10-14T00:00:00Z
+image = "socmedia.jpg"
+weight = 2
+[menu.main]
+parent = "Solutions"
+name = "Apps and social media"
++++
+Today a huge amount of apps and social media allow the user to save, process and share photos and video, but only a few can provide the user with automated suggestions on how to improve that content. At the same time, consumers wish to be able to share their experiences, in the form of photos and videos, with the world.
```
Comment (davidhesselbom) [inline content/solutions/apps.md]: `huge amount`... `huge number`?

Comment (davidhesselbom) [inline content/solutions/apps.md]: `save, process, and share`

**Comment by simonmika** [comment]
Something like: http://www.mobyaffiliates.com/wp-content/uploads/2015/07/screen-shot-2013-10-01-at-12-25-21-pm.png

But that we are actually allowed to use. Either do an image search using that image as a search term and filter by license or take a new photo instead.

**Comment by simonmika** [comment]
I have uploaded a new compression image to [imintsystems/imint.se](https://github.com/imintsystems/imint.se).

**Pull request closed without merging**